code/api/api/src/main/java/com/decathlon/ara/Entities.java

```diff
@@ -17,14 +17,14 @@
 
 package com.decathlon.ara;
 
-import lombok.experimental.UtilityClass;
-
 /**
  * This utility class contains the name of all the entities used in Ara.
  */
-@UtilityClass
```

Comment:
`@UtilityClass` throw exception in the private constructor we can also follow this pattern but it's not really important

Reply:
Pour moi une l'exception n'est pas utile, c'est pour ça que je ne l'ai pas remise